packages/cli/src/Generate.ts

```diff
@@ -228,15 +237,15 @@ ${breakingChangesMessage}`
 This might lead to unexpected behavior.
 Please make sure they have the same version.`
             : ''
-
-        hint = `You can now start using Prisma Client in your code. Reference: ${link('https://pris.ly/d/client')}
+        if (!isDenoDeploy) {
```

Comment:
I think this could be a problem once the preview feature fades away. Is there are reliable way to detect if this is being run inside a Deno project?

Reply:
I think there could be a few options.

- If there's deno.json(c) in the root of the project, then it's deno project
- If there's no `package.json` or `node_modules`, then it's deno project
- If prisma CLI is executed in Deno's [npm compatibility mode](https://deno.com/blog/v1.25#experimental-npm-support), then it's deno project

None of the above are 100% reliable, but we can document what triggers the deno mode.